spec/requests/write_off_forms_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require "rails_helper"
 
-RSpec.describe "WriteOffs", type: :request do
+RSpec.describe "WriteOffForms", type: :request do
   describe "GET /bo/finance_details/:_id/write_off/new" do
```

Comment:
Is this URL still correct?

Reply:
Good catch!! I will change the route path though as I think is better for the user to read this than `/bo/finance_details/:_id/write_off_form/new`